pkg/core/rest/errors/error_handler.go

```diff
@@ -88,6 +89,12 @@ func HandleError(ctx context.Context, response *restful.Response, err error, tit
 				},
 			},
 		}
+	case errors.Is(err, catalog.ErrInstanceNotFound):
+		kumaErr = &types.Error{
+			Status: 404,
+			Title:  "Dataplane not found",
```

Comment:
why dataplane not found? 🤔

Reply:
While testing, I observed that the globalCp instance contains information about the specific global CP instance connected to a zone, which, in turn, has a connection to a DP. I believe the message `instance not found` may be a bit misleading when making a request from the Global CP. In my case, zone was offline while DP was online 🤔